.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Create task on issue and pull request created
+
+on:
+  issues:
+    types: [ opened ]
+  pull_request:
+jobs:
+  addIssueToProjectBoard:
+    runs-on: ubuntu-latest
+    steps:
+    - name: add-new-issues-to-organization-based-project-column
+      uses: docker://takanabe/github-actions-automate-projects:v0.0.1
+      if: github.event_name == 'issues' && github.event.action == 'opened'
+      env:
+        GITHUB_TOKEN: ${{ secrets.repositoryToken }}
```

Comment:
please set this variable to `secrets.REPO_GITHUB_TOKEN`

Reply:
I have corrected this in my recent update.
![pr](https://user-images.githubusercontent.com/45537681/76521224-aa9acb80-6464-11ea-8d5b-52fe3c2166cc.png)